src/constants/typography/secondaryTypography.ts

```diff
@@ -0,0 +1,19 @@
+// TODO: Update when values are finalized
```

Comment:
Keeping the values the same for now, rather than do empty strings like we are for Colors.